aii-opennebula/src/main/perl/opennebula.pm

```diff
@@ -62,18 +62,22 @@ sub make_one
         $rpc = $domainname;
         $main::this_app->info ("Detected configfile RPC section: [$rpc]");
     };
-    my $port = $config->{$rpc}->{port} || 2633;
-    my $host = $config->{$rpc}->{host} || "localhost";
+    my $port = $config->{$rpc}->{port};
+    my $host = $config->{$rpc}->{host};
+    my $url = $config->{$rpc}->{url} || "http://localhost:2633";
     my $user = $config->{$rpc}->{user} || "oneadmin";
     my $password = $config->{$rpc}->{password};
 
+    $url = "http://$host:2633" if ($host);
```

Comment:
@kwaegema ah ok I will change this, in any case the ambiguity disappears due to the opennebula metaconfig aii setup that only allows urls setup: https://github.com/quattor/configuration-modules-core/pull/788